docs/user/supported_devices.rst

```diff
@@ -205,6 +205,7 @@ ath79-generic
 
 * TP-Link
 
+  - Archer C2 (v3)
```

Comment:
Broken devices are also not listed in the docs.

Reply:
Fixed